docs/3-MODELS.md

```diff
@@ -0,0 +1,124 @@
+## 2. MODELS (Storing the collected data)
+
+The data collected from various sources, as discussed in [2-SOURCES](/docs/2-SOURCES.md), is stored in a PostgreSQL
+database. The tables are designed using Django models.
+
+All of the project's models are defined inside the app: `ports`. The models are split into three different files: `port`,
+`buildhistory` and `stats`. Each file contains the models related to the category defined by file name.
+
+#### 1. `Port` Model
+
+It stored the basic information related to a port, with the following fields:
+- `portdir`
+- `description`
+- `homepage`
+- `epoch`
+- `platforms`
+- `categories` - A many-to-many relation with `Category` model
+- `long_description`
+- `version`
+- `revision`
+- `closedmaintainer`
+- `name`
+- `license`
+- `replaced_by` - Default values is `Null` for active ports. If the value is not null, the port is considered as obsolete.
+- `active` - Value is `False` for deleted ports, otherwise `True`*.
+
+***A Port is marked as deleted when the port is available in the app's database, but does not exist in PortIndex. If a 
+port with the same name is added later in time, then previous object is replaced with new object and the newer ports gets
+marked as `active=True` automatically.*
```

Comment:
This way of using kind-of-footnotes is a bit strange (fist you have one star, then two for the actual description; and the star could interfere with markup, ...).
I would have pointed you [here](https://stackoverflow.com/questions/25579868/how-to-add-footnotes-to-github-flavoured-markdown), but if you look at the other document, maybe you could simply add notes at the end of each table (when there's a reason to do so), avoiding footnotes altogether.